mantidproject/mantid
# Update OSIRIS docs

**Description of work.**
This a follow-up PR to #35362 that updates the documentation of the OSIRIS script, adds the release note, and config files example. In addition, this PR fixes a small bug in the empty subtraction scaling. 



**To test:**
- For the docs, build `docs-html` target then check the HTML files.
- For the empty subtraction, try to set the sample_empty_scale in the OSIRIS script to zero.



Fixes #35377. 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
